xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Inputstream.h

```diff
@@ -258,7 +270,7 @@ namespace addon
     * @param enable true for enable, false for disable
     * @remarks
     */
-    virtual void EnableStream(int streamid, bool enable) { }
+    virtual void EnableStream(int streamid, bool enable) = 0;
```

Comment:
I cannot believe, that GetStream and GetStreamId's are NOT used by rtmp,
The risk that the developer searches for why it is not working and its a typo in function name is great.

Pls dont generate code wich is never ever used.